packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1366,6 +1368,50 @@ module.exports = function (log, config, bounces) {
     });
   };
 
+  Mailer.prototype.linkedGoogleAccountEmail = function (message) {
```

Comment:
Lets call this `.postLinkedAccountEmail` to be generic. The calling function will pass in a value `provider` that determines what type of account was linked.

Ref https://github.com/mozilla/fxa/blob/9fd866c8cf02c658d4cebd4e34a4c4dd9589e817/packages/fxa-shared/db/models/auth/linked-account.ts#L8